samtranslator/model/connector/connector.py

```diff
@@ -20,6 +20,11 @@
     ],
 )
 
+STANDARD_CONNECTOR = "StandardConnector"
+EMBEDDED_CONNECTOR = "EmbeddedConnector"
+MULTIDEST_CONNECTOR = "MultiDestConnector"
+EMBEDDED_MULTIDEST_CONNECTOR = "EmbeddedMultiDestConnector"
```

Comment:
Why a separate type? Instead of individual booleans, and then for metrics can check combination. Otherwise with more kinds of connectors this will grow exponentially. Also should keep code simpler.

Reply:
great ideas